src/sentry/models/outbox.py

```diff
@@ -458,6 +459,15 @@ def find_scheduled_shards(cls, low: int = 0, hi: int | None = None) -> List[Mapp
     def prepare_next_from_shard(cls, row: Mapping[str, Any]) -> Self | None:
         using = router.db_for_write(cls)
         try:
+            logging.info(
```

Comment:
You'll be logging on the root logger here. Usually we make a locally scoped logger with `logger = logging.getLogger(__name__)`.